pyMultiOmics/query.py

```diff
@@ -81,9 +81,13 @@ def run(self, pipeline, _):
         mapping = pipeline.mapping
         data = []
         for node_id in self.node_ids:
-            node_data = self._get_node_data(mapping, node_id)
-            row = self._to_row(node_data)
-            data.append(row)
+            try:
+                node_data = self._get_node_data(mapping, node_id)
+                row = self._to_row(node_data)
+                data.append(row)
+                logger.info("Mapping was achieved for %s " % node_id)
```

Comment:
Let's remove this line or change the log level to debug, so we don't print too many things.

Reply:
Logger line is now removed from the code.